variables.tf

```
@@ -101,6 +101,24 @@ variable "network_rule_set_trusted_services_allowed" {
   nullable    = false
 }
 
+variable "queue" {
+  description = "Manages a ServiceBus Queue."
+  type = map(object({
+    name                 = string
+    partitioning_enabled = optional(bool, false)
+  }))
+  default = {}
+}
+
+variable "topic" {
+  description = "Manages a ServiceBus Topic."
```

Name should be plural since value type is a collection:

```suggestion
variable "topics" {
  description = "A map of topics to create for this Service Bus namespace. Only applicable if value of SKU is \"Standard\" or \"Premium\"."
```

Also updated description to be more in line with how we usually write descriptions.

Default object property values look good to me 😊

---

References to this variable must be updated to reference this new name as well.